src/web/handlers/metadata.py

```diff
@@ -0,0 +1,80 @@
+"""
+API handler for MyGeneset submit/ endpoint
+"""
+
+
+import elasticsearch
+import asyncio
+from elasticsearch import Elasticsearch
+from biothings.web.handlers import MetadataSourceHandler
+from tornado.web import HTTPError
+
+class MyGenesetMetadataSourceHandler(MetadataSourceHandler):
+    """"
+    Handler for GET /metadata
+
+    {
+        ...
+        "stats": {
+            "total": 10,
+            "curated": 6,
+            "user": 4,
+            "anonymous": 2
+        }
+        ...
+    }
+    """
+
+    async def _count_curated(self):
+        """Fetch a geneset document from Elasticsearch"""
+        try:
+            document = await self.biothings.elasticsearch.async_client.count(
+                    index=self.biothings.config.ES_CURATED_INDEX)
+        except elasticsearch.exceptions.NotFoundError:
+            raise HTTPError(404, None, "_count_curated", reason="Document does not exist.")
```

Comment:
* when index is not found, I think it's reasonable to return the count as 0.
* `elasticsearch.exceptions.NotFoundError`: since only `NotFoundError` is used from `elasticsearch`, you may just do `from elasticsearch.exceptions import NotFoundError` at the top.

Same comments for the cases below too.